### PR TITLE
FIX BACKLIGHT -fix #69

### LINKED ISSUE #69: BACKLIGHT can't stay in OFF mode
**Describe the bug**
As soon as an LCD update is carried out, the backlight is automatically activated. So when I do lcd_backlight(0), it only works for one update.

**Executed command and associated error**
n.d

**Host and software info**
- RPi board version: `RPi 4` `RPi 3`

**Checklist**
- [x] I have watched and followed the instructions in the [Youtube tutorial](https://www.youtube.com/watch?v=fR5XhHYzUK0).
- [x] I have searched [open and closed issues](https://github.com/the-raspberry-pi-guy/lcd/issues) for either an identical or similar bug before reporting this new one.

**Additional context**
n.d

### PR DESCRIPTION
**Pull request by Jumitti**
The BACKLIGHT function is very useful but it has a bug. Each update of the LCD screen resets the BACKLIGHT value to ON. So we cannot have a permanent display without backlight. I fixed the problem. The default display is with backlight. You can configure this with lcd_backlight as before. If you do lcd_blacklight(1) then it will be lit. If it is 0 it will be turned off. I added security, if you put any number other than 1 or 0 then it will necessarily be on.

There are a lot of commits but because I'm also working on another project and I'm doing a lot of testing, sorry.

### CODE REVIEW

File: .gitignore
Comment (cgomesu) [inline .gitignore:1]: this file is unrelated to the issue. not sure why it was modified.

File: drivers/i2c_dev.py
Comment (cgomesu) [inline drivers/i2c_dev.py:1]: the changes look reasonable but did you check whether this change breaks any other demo? this needs to be done.

**Review by cgomesu — CHANGES_REQUESTED** [review]
I left a couple of comments in the changed files. Please go over each one of them before making any additional changes. 

> There are a lot of commits but because I'm also working on another project and I'm doing a lot of testing, sorry

I noticed that, too. As a general rule of thumb, create (and use) a new branch **from the master one** for each meaningful change. Unfortunately, there are too many commits here that have nothing to do with the issue and this will currently prevent me from merging the changes to the main branch.

There are a couple of options for you here. You can either try to edit the commit history, so that there's a single commit for backlight issue in the `i2c_dev.py` file, or startover: drop this PR; on you local repo; checkout into the master branch, make sure it is up-to-date; create a new branch from it to fix the backlight (e.g., `hotifx/backlight`); checkout into the new branch, make the changes and commit them; write a new PR and mention this one there (e.g., `- redo of #70`). Please notice that if you choose to startover, you still need to address the commends I left the files you changed here.

**Comment by Jumitti** [comment]
Sorry, I use github just for myself usually. At first, I just wanted to make a PR then reading the guidelines I saw that I also had to make an Issue. But I don't know how to link PR and Issue.

I will test if the other scripts work well with it too. For now, I know that with my script everything works very well but it's worth a look for others

File: .gitignore
Comment (Jumitti) [reply .gitignore:1]: Sorry, I'm working on a bigger script where I use APIs. And in the .gitignore I put my test script and also my .json with access to the API. I created a new branch for the backlight fix and I modified the .gitignore so that it is identical

File: drivers/i2c_dev.py
Comment (Jumitti) [reply drivers/i2c_dev.py:1]: Sorry, I use github just for myself usually. At first, I just wanted to make a PR then reading the guidelines I saw that I also had to make an Issue. But I don't know how to link PR and Issue.

I will test if the other scripts work well with it too. For now, I know that with my script everything works very well but it's worth a look for others
